app/grants/models.py

```diff
@@ -375,6 +375,12 @@ class Meta:
 
     funding_info = models.CharField(default='', blank=True, null=True, max_length=255, help_text=_('Is this grant VC funded?'))
 
+    clr_prediction_curve = ArrayField(
+        ArrayField(
+            models.FloatField(),
+            size=2,
+        ), blank=True, default=list, help_text=_('5 point curve to predict CLR donations.'))
```

Comment:
^ This works for now -> but curious if you've given any thought on what the this would be the total accumulated match irrespective of chains / would we wanna move this to a table by itself which has 
`grant_fk` | `clr_round_fk` | `clr_prediction_curve` which would store finer details ?

Grant A is part of Zcash Tech & Eth Tech 
`clr_prediction_curve` -> would be a property which show the total prediction match across all active rounds/chains

and the new table would would have 2 entries in fk table 
`grant_fk` | `clr_round_fk` | `clr_prediction_curve` with clr_prediction_curve for both Zcash & EthTech

Reply:
> the this would be the total accumulated match irrespective of chains / would we wanna move this to a table by itself which 

not sure i totally understand the question (typo in the middle?) but the GrantCLRCalculation basically hadnles this i think

i think if we want to extend the match estimates to only look at one (or a subset of) of curves, the existing FK structure handles that.  i dont see a strong reason to store it in the grant-level one